ocaml/xapi/quicktest_cbt.ml

```diff
@@ -74,21 +95,27 @@ let vdi_data_destroy_test ~session_id ~vDI =
     VDI.enable_cbt ~session_id ~rpc:!rpc ~self:vDI;
     assert_cbt_status true ~session_id ~test ~vDI
       ~msg:"VDI.enable_cbt failed";
+    test_vdi_update ~session_id ~test ~vDI;
 
     debug test "Snapshotting original VDI with CBT enabled";
     let snapshot = VDI.snapshot ~session_id ~rpc:!rpc ~vdi:vDI ~driver_params:[] in
     assert_cbt_status true ~session_id ~test ~vDI:snapshot
       ~msg:"VDI.snapshot failed, cbt_enabled field didn't carry over";
+    test_vdi_update ~session_id ~test ~vDI;
```

Comment:
See my comment below on line 118 - if we want to follow the same pattern as there, we should check both `snapshot` and `vDI` here.